### ISSUE TITLE
Add support in isAuthorzed for MNodes, not just CNodes

### ISSUE THREAD

**Issue by amoeba** [DataONEorg/rdataone#158]
I expected this to work and not produce an error:

```{r}
 class(mn)
[1] "MNode"
attr(,"package")
[1] "dataone"
> dataone::isAuthorized(mn, id = metadata_pid, "write")
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘isAuthorized’ for signature ‘"MNode"’
```

[The docs](http://jenkins-1.dataone.org/jenkins/job/API%20Documentation%20-%20trunk/ws/api-documentation/build/html/apis/MN_APIs.html) say MNodes also support this method. Could support be added so I can ask an MN and not just a CN?

**Comment by gothub** [comment]
@amoeba Yep, this was an omission - I'll add isAuthorized for `MNode`.

**Comment by amoeba** [comment]
Great, thanks! Lemme know when you do and I'll give it a whirl.

**Comment by gothub** [comment]
`isAuthorized` now works for MNode or CNode instances. Fixed in commit afbe041c46b3e4029dbf17d373ea8fcd86c2fa3d
